inception-project/inception
# #2027 - PDFs do not load with reverse proxy + https

**What's in the PR**
- Calculate the callback URLs relative to the URL of the iframe in which pdf.js is loaded

**How to test manually**
* Run INCEpTION behind a https reverse proxy which forwards to http
* Check if simply opening a PDF for annotation on Windows works

**Automatic testing**

**Documentation**
